### PR TITLE
Small improvements to README

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Capitalize proper noun GitHub
- Use mono-spaced font for some defaults
- Use actions/checkout@v2 in example as that is the latest stable release
- Improve grammar of some sentences

## Context:

I saw some small things that I could improve. 😉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,18 +20,18 @@ This action will create a github release and optionally upload an artifact to it
...
+- **token**: (**Required**) The GitHub token. Typically this will be `${{ secrets.GITHUB_TOKEN }}`.
 
 ## Action Outputs
 - **id**: The identifier of the created release.
 - **html_url**: The HTML URL of the release.
 - **upload_url**: The URL for uploading assets to the release.
 
 ## Example
-This example will create a release when tag is pushed:
+This example will create a release when a new tag is pushed:
```
Comment (ncipollo) [inline README.md:34]: Could we revert this line? The workflow in this example will actually run when any update to a tag is pushed, whether it be a new tag or an existing tag.

**Review by ncipollo — CHANGES_REQUESTED** [review]
Thanks for the documentation update! Just one tiny tweak then this should be ready to merge!

File: README.md (hunk 1 above)
Comment (HonkingGoose) [reply README.md:34]:
```suggestion
This example will create a release when a tag is pushed:
```

**Comment by ncipollo** [comment]
Looks good, thanks for the pull request!

**Pull request merged**